commercial-paper/organization/digibank/configuration/cli/docker-compose.yml

```diff
@@ -29,6 +29,7 @@ services:
     volumes:
         - /var/run/:/host/var/run/
         - ./../../../../organization/digibank:/opt/gopath/src/github.com/
+        - ./../../../../organization/digibank:/opt/gopath/src/github.com/hyperledger/fabric-samples/commercial-paper/organization/digibank
```

Comment:
I'd be careful about having overlapping volume mounts here.

Reply:
Removed older mount